lightningd/plugin.c

```diff
@@ -1007,3 +1057,27 @@ void json_add_opt_plugins(struct json_stream *response,
 		json_add_string(response, "plugin", p->cmd);
 	}
 }
+
+/**
+ * Determine whethe a plugin is subscribed to a given topic/method.
+ */
+static bool plugin_subscriptions_contains(struct plugin *plugin,
+					  const char *method)
+{
+	for (size_t i = 0; i < tal_count(plugin->subscriptions); i++)
+		if (streq(method, plugin->subscriptions[i]))
+			return true;
+
+	return false;
+}
+
+void plugins_notify(struct plugins *plugins,
```

Comment:
LGTM ( I was searching if subscriptions to the same events would work if several plugins subscribed to the same event

Reply:
That is indeed supported, since unlike jsonrpc calls and hooks there is no problem with notification semantics.